.github/workflows/release.yml

```diff
@@ -8,8 +8,11 @@ jobs:
     name: release
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-node@v1
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version-file: '.nvmrc'
```

Comment:
instead of using `nvmrc`, why not use the `node-version` parameter and set it to 16?

Reply:
By using the `.nvmrc` file, when we want to update the node version we just need to change the `.nvmrc` file and it will be used for both local development and CI/CD pipelines automatically.

Reply:
I'm trying to avoid dot files when possible, but I don't mind, we can ship it as is and change it later if it causes any problems